### PR TITLE
All Functionalities implemented

### PR DESCRIPTION
**Pull request by ricricucit**

### CODE REVIEW

File: src/store.js [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
 import Vue from 'vue'
 import { AeternityClient } from '@aeternity/aepp-sdk'
+
 // import { reject } from 'any-promise'
 
 Vue.use(Vuex)
 
 const store = new Vuex.Store({
   state: {
-    access_key: '8ytPIx43o8QSvYCGDu8hkllxAU9gnZr6HByqnvH8PRa2UANhwl',
```
Comment (muxe) [inline src/store.js]: the history still contains the secret

**Comment by muxe** [comment]
Now that I commented I cannot accept this PR anymore.

This PR is too large for me to really review thoroughly but I ran it locally, connected the scanner and it looked fine. ;)

File: src/components/Scanner.vue [hunk 2]
```diff
@@ -62,41 +82,79 @@ export default {
     }
   },
   methods: {
+    cutHashes (hash, cutStart, cutEnd) {
+      return hash.substr(0, cutStart) + '...' + hash.substr(cutEnd + 1)
+    },
+    printDate (ts) {
+      const date = new Date(ts)
+      return `${String(date.getDate()).padStart(2, '0')}/${String(date.getMonth()).padStart(2, '0')}/${date.getFullYear()} – ${String(date.getHours()).padStart(2, '0')}:${String(date.getMinutes()).padStart(2, '0')}`
```
Comment (outergod) [inline src/components/Scanner.vue:90]: Uuuuuuugh

File: src/store.js [hunk 3]
```diff
@@ -1,64 +1,46 @@
...
   state: {
-    access_key: 'secret!',
-    barState: 'open',
+    access_key: '',
+    barState: 'open', // open, closed, out_of_beer
     account: {
-      pub: 'ak$3evGruG5reEY4eWDKCuZxkDBp4KTRyj4YJp98BGTgSegqURNpaTs2FEzVxHbiZwA4Z48JatQzNBoZEGM732BwDRhz3Ng3U',
-      priv: '94e013ac4089d5002ccf6b4807952f3bd387540ce2af5d66e5490760a7086b85',
+      pub: 'ak$3TRJBCvcvaegewQkexWVQkt7bEFf1tCvhvj6jfErZQNWyJ4NoyxUwkGrVVWDefxPpPEiY534fTutPaURn72HrGKCYaNWPM',
+      priv: '',
```
Comment (outergod) [inline src/store.js:15]: Too late, but we can rewrite history before making it public. Actually, we *must* do it

**Review by outergod — APPROVED** [review]

**Pull request merged**